book/001-Introduction.md

```diff
@@ -40,11 +40,11 @@ If you know:
 
 You should be alright.
 
-If you feel moderately comfortable with a popular modern framework like `React`, `Angular 2+`, or `Vue.js`, you should be in pretty good shape too. We'll use as many real world examples with these frameworks throughout this book, not just the front end UI stuff, but backend Node scripts and servers as well.
+If you feel moderately comfortable with a popular modern framework like `React`, `Angular 2+`, or `Vue.js`, you should be in pretty good shape, too. We'll use as many real world examples with these frameworks throughout this book, not just the front end UI stuff, but backend Node scripts and servers as well.
 
 # Navigating the Book
 
-Like many other developers, my eyes immediately move towards snippets of code. This book attempts to show as many examples as possible and keep the non-coding chat to a minimum. All important topics are in header tags for easy maneuvering and link-ability for sending information to colleagues and coworkers.
+Like many other developers my eyes immediately move towards snippets of code. This book attempts to show as many examples as possible and keep the non-coding chat to a minimum. All important topics are in header tags for easy maneuvering and link-ability for sending information to colleagues and coworkers.
```

Comment:
I don't think this change is necessary. The comma is an implied pause and creates a compound sentence that's reversed:

`My eyes immediately move towards snippets of code, like many other developers`

Reversing this sentence, you'll need a comma. I'm no English expert, but if you can add that comma back in, I'd really appreciate it :).